pkg/kubelet/kubelet.go

```diff
@@ -2893,6 +2897,7 @@ func (kl *Kubelet) updateRuntimeUp() {
 		kl.runtimeState.setRuntimeState(fmt.Errorf("container runtime not ready: %v", runtimeReady))
 		return
 	}
+	kl.runtimeHandlers = s.Handlers
```

Comment:
one nit: it seems more idiomatic to use a setter here instead of directly setting the field. Can you do that?

Reply:
changed in the last version